app/src/main/res/values/strings.xml

```diff
@@ -123,6 +123,7 @@
   <string name="patients_scan_simple_card">Scan Simple Card</string>
   <string name="patients_recentpatients_title">Recent Patients</string>
   <string name="patients_recentpatients_no_recent_patients">No recent patients</string>
+  <string name="patients_recentpatients_see_all">SEE ALL</string>
```

Comment:
Can we not use full capitalization in the string resources? Since these are buttons, we should be defining these buttons in title case and then use `textAppCaps=true` on the button/textview to capitalize them.

Reply:
I'll make this change and push.